src/renderer/entities/staking/lib/staking-utils.ts

```diff
@@ -0,0 +1,9 @@
+import type { ChainId } from '@shared/core';
+
+export const stakingUtils = {
+  isKusamaChainId,
+};
+
+function isKusamaChainId(chainId: ChainId): boolean {
```

Comment:
Why do we need that?

Reply:
It's used here, temporary fix for Kusama's nominators
```javascript
function getDefaultValidatorsAmount(api: ApiPromise): number {
  if (stakingUtils.isKusamaChainId(api.genesisHash.toHex())) return KUSAMA_MAX_NOMINATORS;

  return DEFAULT_MAX_NOMINATORS;
}
```